packages/react-native-session-replay/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@datadog/mobile-react-native-session-replay",
-    "version": "2.3.2-alpha.0",
+    "version": "2.3.3-alpha.0",
```

Comment:
doesn't match the version in `Podfile.lock` (I guess podfile should still reference alpha)

Reply:
It was an imprecision caused by the `update-versions.sh` script